config/config.go

```diff
@@ -46,14 +47,11 @@ type configModel struct {
 
 // DBConf holds all DB related configurations
 type DBConf struct {
-	Protocol       string `json:"Protocol"`
-	Host           string `json:"Host"`
-	Port           string `json:"Port"`
-	MinIdleConns   int    `json:"MinIdleConns"`
-	PoolSize       int    `json:"PoolSize"`
-	RedisHAEnabled bool   `json:"RedisHAEnabled"`
-	SentinelPort   string `json:"SentinelPort"`
-	MasterSet      string `json:"MasterSet"`
+	Protocol           string `json:"Protocol"`
+	Address            string `json:"Address"`
+	MinIdleConns       int    `json:"MinIdleConns"`
```

Comment:
Why are we dumping the port configuration?

Reply:
Isn't better to keep just one field for the address instead of collecting host and port separately and joining them later in code? Seen this approach in URP.